distro/common/knot.service

```diff
@@ -10,6 +10,49 @@ User=knot
 Group=knot
 CapabilityBoundingSet=CAP_NET_BIND_SERVICE CAP_SETPCAP
 AmbientCapabilities=CAP_NET_BIND_SERVICE CAP_SETPCAP
+# Require systemd >= v238
+#TemporaryFileSystem=/run:ro /var:ro
+# Require systemd >= v233
+#BindPaths=/run/systemd
```

Comment:
I'm curious, why is `/run/systemd` needed?

Reply:
`knot` use `sd_notify(0, "READY=1");` to tell `systemd` when it ready, for that to work it need access to `/run/systemd`.
It could be more specific though (`/run/systemd/notify`), but I haven't tested if systemd is used for anything else.

Reply:
I see logging: `sd_booted()` and `sd_journal_send()`.

Reply:
`sd_booted()` check if the dir `/run/systemd/system/` and `sd_journal_send()` writes to `/run/systemd/journal/socket`.

So I think `BindPaths=/run/systemd` is the best solution.